suites/create.js

```diff
@@ -78,37 +78,46 @@ export function runDataIntegrityProofFormatTests({
         }
       }
     });
-    it(`"proof.type" field MUST be "${expectedProofTypes.join(',')}" ` +
-      `and the associated document MUST include expected contexts.`,
-    function() {
-      for(const proof of proofs) {
-        proof.should.have.property('type');
-        proof.type.should.be.a(
-          'string',
-          'Expected "proof.type" to be a string.'
-        );
-        const hasExpectedType = expectedProofTypes.includes(proof.type);
-        hasExpectedType.should.equal(true);
-
-        if(proof.type === 'DataIntegrityProof') {
-          const expectedContexts = [
-            'https://www.w3.org/ns/credentials/v2',
-            'https://w3id.org/security/data-integrity/v2'
-          ];
-          const hasExpectedContexts = expectedContexts.some(
-            value => data['@context'].includes(value));
-          hasExpectedContexts.should.equal(true);
-        }
+    if(expectedProofTypes.includes('DataIntegrityProof')) {
```

Comment:
Is the `expectedProofTypes` variable coming from whichever DI test suite calls these assertions?

Reply:
here is the function in question: https://github.com/w3c-ccg/data-integrity-test-suite-assertion/blob/a701beaf6443e7f233f142d143bdefed168ae279/index.js#L31-L54

Just so this is known the functions in found in the suites files are not exported directly, but rather have another layer inside of index.js that does some minor formatting and keyType related stuff that will need to be refactored at another time.